test/feeds.test.js

```diff
@@ -62,3 +62,55 @@ describe('test /feeds/:id responses', () => {
     expect(res.body instanceof Array).toBe(false);
   });
 });
+
+// https://github.com/Seneca-CDOT/telescope/issues/734 Tests below will need to have authorization to pass.
+describe('test POST /feeds endpoint', () => {
+  it('responds with json', async () => {
+    const feedData = {
+      author: 'foo',
+      url: 'http://telescope200.cdot.systems',
+    };
+    const res = await request(app)
+      .post('/feeds')
+      .send(feedData)
+      .set('Accept', 'application/json');
+    expect(res.status).toEqual(201);
+  });
```

Comment:
Let's modify this test a bit.  I care less about it responding with JSON, and more that it gives back a usable id.  After your code above, do this:

```js
// See if the feed was created as expected.
const id = res.body.id;
const feed = await Feed.byId(id);
expect(feed.author).toEqual(feedData.author);
expect(feed.url).toEqual(feedData.url);
```

This code is untested, but hopefully it gives you enough.